### DISCUSSION TITLE
404: Requested url does not match any rules

### DISCUSSION THREAD

**Discussion by usmann123706** [ory/oathkeeper#956] [Q&A] [answered]
Hi,
I am getting the following error
{"error":{"code":404,"status":"Not Found","message":"Requested url does not match any rules"}}
My rules are

[
  {
    "id": "allow-anonymous-with-header-mutator",
    "version": "v0.38.23-beta.1",
    "upstream": {
    
	  "url": "http://172.16.0.4/Login3.aspx"
    },
    "match": {
      "url": "http://localhrulesost:4455/",
      "methods": [
        "GET"
      ]
    },
    "authenticators": [
      {
        "handler": "anonymous"
      }
    ],
    "authorizer": {
      "handler": "allow"
    },
    "mutators": [
      {
        "handler": "header",
        "config": {
          "headers": {
            "X-User": "{{ print .Subject }}"
          }
        }
      }
    ]
  }
]

 Kindly assist in this regard
 Thanks

**Comment by vinckr** [comment, marked as answer]
See this example: https://github.com/gen1us2k/ory_workshop/tree/master/oathkeeper

To echo my colleagues request in the chat: 

Could you please tell us more about your usecase? What are you trying to achieve? 
What’s the current configuration and how are you going to use Ory Oathkeeper with your app?

**Reply by vinckr** [reply]
Hey @usmann123706 , just checking in if your problem still persists. 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
